README.md

```diff
@@ -21,12 +24,22 @@ One common setup is to create a directory for a manuscript that contains both th
 Under this setup, you can run the Manubot using:
 
 ```sh
-manubot \
+manubot process \
   --content-directory=content \
   --output-directory=output
 ```
 
-See `manubot --help` for documentation of all command line arguments.
+See `manubot process --help` for documentation of all command line arguments.
+
+### Cite
+
+`manubot cite` is a command line utility to create CSL JSON items for one or more citations.
```

Comment:
This could be a good place to link `CSL JSON` to more information about the format.